tutorials/embedded/step-1.html.haml

```diff
@@ -42,7 +42,7 @@ title: Embedded Step-by-Step Tutorial
         %code.java
           = preserve do
             :escaped
-              CacheManager cacheManager = new DefaultCacheManager();
```

Comment:
[Here](https://github.com/diegolovison/example-infinispan/blob/b7bf0faca266249761d9f381e1ee42581a40d986/src/main/java/com/github/diegolovison/example/infinispan/EmbeddedCacheManagerExample.java#L24) is the example